src/content/autoBlock.ts

```diff
@@ -118,6 +118,13 @@ interface User {
 	alreadyBlocked: boolean;
 }
 
+function blockEthUsername(userName?: string) {
+	console.log(userName);
```

Comment:
Remove console.log

Reply:
Remove the settings check from here, it belongs in the callback with the rest

Reply:
lol oops - should go unblock a few elizabeths, bethanys, and beths 😂